src/main/java/net/glowstone/net/message/play/player/UseItemMessage.java

```diff
@@ -2,9 +2,14 @@
 
 import com.flowpowered.network.Message;
 import lombok.Data;
+import org.bukkit.inventory.EquipmentSlot;
 
 @Data
 public class UseItemMessage implements Message {
 
     private final int hand;
+
+    public EquipmentSlot getEquipmentSlot() {
+        return EquipmentSlot.values()[hand];
```

Comment:
I would do this instead: https://github.com/GlowstoneMC/Glowstone/blob/master/src/main/java/net/glowstone/net/message/play/player/BlockPlacementMessage.java#L14

That way if the client sends an incorrect hand ID, it will default to the main hand.